addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11445,7 +11445,13 @@ msgctxt "#19346"
 msgid "Display and manage available PVR client add-ons."
 msgstr ""
 
-#empty strings from id 19347 to 19498
+#. info message box text stating that none of the available pvr clients does provide client-specific settings
+#: xbmc/pvr/PVRGUIActions.cpp
+msgctxt "#19347"
+msgid "None of the active PVR clients does provide client-specific settings."
```

Comment:
You can drop the auxiliary verb and go third-person plural: "None of the active PVR clients provide client-specific settings."

Reply:
Thanks, will be changed.